src/main/ts/Part2Ex4FP.ts

```diff
@@ -143,7 +143,7 @@ const constant = <A> (a: A) => (...args: unknown[]): A => a;
 const always3 = constant(3);
 
 /*
-So, constant ignores whatever is passed for the B parameter, and just returns the A.
+So, constant ignores whatever are passed for input parameters, and just returns the A.
```

Comment:
```suggestion
So, constant ignores whatever is passed for input parameters, and just returns the A.
```
Just a minor grammar fix.